hooks/pre-exit

```diff
@@ -12,7 +12,10 @@ df -h /tmp/
 # Remove tempdir
 case "$BUILDKITE_AGENT_META_DATA_QUEUE" in
     "new-central")
-        srun --ntasks-per-node=1 --ntasks=${SLURM_JOB_NUM_NODES} rm -rf "/tmp/slurm_${SLURM_JOB_ID}"
+        (pdsh -w "$SLURM_NODELIST" rm -rf "/tmp/slurm_${SLURM_JOB_ID}") || { 
```

Comment:
I am surprised parenthesis work here

Reply:
It worked when I tested it in an interactive slurm job, but the parentheses are definitely not needed.